features/environment.py

```diff
@@ -35,5 +40,11 @@ def before_all(context):
     context.authorized_page = AuthorizedPage(context)
 
 
+def after_scenario(context, scenario):
+    if scenario.status == "failed":
+        context.driver.save_screenshot('scenario_result/'+scenario.name + get_date_time() + "_failed.png")
```

Comment:
could you also commit "scenario_result" directory to this PR please?

Reply:
Already added.